src/components/PeopleManagement/GroupCardGrid.jsx

```diff
@@ -33,6 +33,7 @@ const GroupCardGrid = ({ groups }) => {
         <Collapsible
```

Comment:
An alternative (preferred) would be to switch to using `Collapsible.Advanced` so it's possible to use CSS utility classes on the `Collapsible.Trigger` sub-component. This approach would be preferred as it doesn't involve overriding Paragon CSS (anti-pattern).

Example: [Playground](https://paragon-openedx.netlify.app/playground?code=N4Igxg9gJgpiBcIA8BhCAbdBDADgZwEsAjdGAOgEEoA3LAOzBigD4AdOgAg9Q232NJkAKgCcCAc3EwRHMNjx4AclgC2MALysQUALQAzUgA8OAKwCueAC4E9ATx2Q6lmE50uoWtpy4cAygAsIAHcOLEwORQ5xEQgzfHYuJAB6NExcQhJyUQkpES9E1L4MwQAhaFt8nyQcZgBNWJkrMz09KIgYPA5-aRhkmoTuFN50gXIyqAr2ZMKRzMoaekYWdhAAGhBLbrU8BABtECYADTIIEXEQAF11oIIoTZ34XYB2ADYADguAXyA)

```jsx
<Collapsible.Advanced>
  <Collapsible.Trigger className="d-flex justify-content-end">
    Toggle Collapsible
  </Collapsible.Trigger>
  <Collapsible.Body>
    <p>Your stuff goes here</p>
  </Collapsible.Body>
</Collapsible.Advanced>
```

That said, makes sense to roll out the current proposed fix as a sooner remedy.